core/src/main/java/org/ocpsoft/prettytime/ResourcesChronoUnit.java

```diff
@@ -0,0 +1,102 @@
+package org.ocpsoft.prettytime;
+
+import org.ocpsoft.prettytime.units.*;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+
+public enum ResourcesChronoUnit {
+    JUST_NOW(ChronoUnit.SECONDS.getDuration(), "JustNow"),
```

Comment:
I think this will need to be corrected as `JUST_NOW` is not a 1:1 correlation to `ChronoUnit.SECONDS`. I believe `JustNow` is actually "60 seconds or less".

See [JustNow](https://github.com/ocpsoft/prettytime/pull/249/files#diff-32daade17eed2e6a8615e927576111fbf8f8e84e36b929e1da7b9aa71dc48123R29):

```
setMaxQuantity(1000L * 60L);
```